src/state/reducers/buddies.ts

```diff
@@ -68,6 +68,24 @@ export const reducer: automaton.Reducer<State, actions.Action> = (
         ),
       );
 
+    case 'buddies/changeBanStatusBatch/start':
+      return automaton.loop(
+        RD.pending,
+        withToken(
+          buddyApi.banBuddies(
+            action.payload.buddyIds,
+            action.payload.banStatus,
+          ),
+          actions.make('buddies/changeBanStatusBatch/end'),
+        ),
+      );
+
+    case 'buddies/changeBanStatusBatch/end':
+      return automaton.loop(
+        RD.pending,
+        withToken(buddyApi.fetchBuddies, actions.make('buddies/completed')),
+      );
+
```

Comment:
So this means that when the batchDelete is done, then afterwards the buddies are fetched again?
I.e. the banBatchRequest does not give the updated buddyResponse?

Reply:
Yep. The batchBanRequest returns the updated response but it is not used right now for modifying the state directly as this was simpler for me to implement (and understand).

Reply:
Ok yeah. Imo merge this one as it is, and then I could refactor a bit into #151